### DISCUSSION TITLE
Disable pagination using breakpoints

### DISCUSSION THREAD

**Discussion by Rapid0s** [nolimits4web/swiper#5449] [Q&A]
I am trying to use breakpoints in order to show and hide pagination bullets but with no luck.
`		breakpoints: {
			0: {
				pagination: {
					el: '.swiper-pagination',
					clickable: true,
					type: 'bullets',
				}
			},
			991: {
				pagination: false		 
			}
		}
`
If I start under 991 width bullets are created and event if I resize over 991 works like expected by hiding them.
if I start over 991 pagination bullets are not created. 

Any ideas ?

**Comment by Arhell** [comment]
if you do not want to see pagination, you can remove it using css

**Reply by Rapid0s** [reply]
Yeah, I ended up creating pagination for all sizes and hiding it with responsive rules
